clients/cli/src/main/java/org/apache/gravitino/cli/commands/CatalogDetails.java

```diff
@@ -35,14 +36,17 @@ public class CatalogDetails extends Command {
    *
    * @param url The URL of the Gravitino server.
    * @param ignoreVersions If true don't check the client/server versions match.
-   * @param outputFormat The output format.
+   * @param listOptions The list options.
```

Comment:
The name of this variable is confusing.

Reply:
Reverted to using outputFormat for @param.